src/builders/default.nix

```diff
@@ -21,8 +21,10 @@
   };
   
   rust = rec {
-    default = buildRustPackage;
+    default = crane;
```

Comment:
Should the default be made crane?

Reply:
I leave that decision up to you.

Reply:
I made it the default builder since it allows for more caching than `buildRustPackage`. It should be a sane default if we fix the IFD woes. It also seemed much simpler to me than how `buildRustPackage` reads / works, so I think it would be more easy to override.